Sources/GraphQLTransportWS/Client.swift

```diff
@@ -139,7 +141,35 @@ public class Client<InitPayload: Equatable & Codable> {
             ).toJSON(encoder)
         )
     }
-    
+
+    public func addObservable(observable: Observable<EventLoopFuture<GraphQLRequest>>) {
+        observable.subscribe(
+            onNext: { [weak self] resultFuture in
+                guard let self = self else { return }
+                resultFuture.whenSuccess { request in
+                    self.sendNext(payload: request, id: UUID().uuidString)
+                }
+                resultFuture.whenFailure { error in
+                    self.error(.graphQLError(error))
+                }
+            }
+        ).disposed(by: disposeBag)
+    }
+
+    /// Send a `next` message through the messenger.
+    /// - Parameters:
+    ///   - payload: `GraphQLRequest` object for the server to handle
+    ///   - id: id of the message
+    private func sendNext(payload: GraphQLRequest, id: String) {
```

Comment:
Does the server send a response back to the client that is the result of running this GraphQLRequest, or does it swallow it? If it sends one back, how do we identify this response vs a normal subscription `next` event?

Perhaps making new message types (like `InSubQuery` and `InSubResponse` or something) would help clarify which messages are which.

Also some example workflows like [the ones in graphql-transport-ws](https://github.com/enisdenjo/graphql-ws/blob/master/PROTOCOL.md#examples) would clarify how this feature can be used.

Reply:
From talking in person:
We should send a `GraphQLResponse` instead of a `GraphQLRequest` as next frames, and only do so if we are within an already initialized active subscription.

Reply:
Also added a similar example workflow to the README

Reply:
Awesome, great docs! Thanks!